lhotse/dataset/transforms/cat.py

```diff
@@ -0,0 +1,79 @@
+from typing import Optional, Sequence
+
```

Comment:
I think calling this concatenate rather than cat would be clearer (filename and class name), but up to you.

Reply:
Does "pack" sound better maybe? Trying to avoid long names (also found cutcat funny somehow but clear would be indeed better)

Reply:
I think concatenate is clearer than pack.